Yoast/wordpress-seo
# Remove the monorepo version bumps from the RC automatization

The monorepo versions are already bumped when the monorepo RC's are created

## Context


* It used to be the case that when creating an RC for Free, we would check whether the monorepo RC versions had been correctly bumped in the Free code. However, it is part of the monorepo RC creation process to bump those versions in Free. Together with @IreneStr it was decided that the monorepo RC version check/bump can be left out. This version bump in Free probably even led to a bug when Free RC's were created the day before release, when the monorepo RC's for the _next_ release had already been created.

## Summary


This PR can be summarized in the following changelog entry:

* Removes the monorepo version bumps from the RC automatization.

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

* Create an RC [with automation](https://yoast.atlassian.net/wiki/spaces/PLUGIN/pages/282558472/Yoast+SEO+Free+creating+RCs+using+automation). 
* There should _not_ be a prompt to verify whether the versions for the javascript monorepo packages are correctly set.
* If you make it to the stage where you should enter the changelog, you can break off the script.
* Inspect the pushes that the script made to GitHub, there should be nothing related to monorepo RC versions.

## UI changes

## Documentation

## Quality assurance


Fixes #
